codegen/generated!.gotpl

```diff
@@ -268,14 +274,14 @@
 		if ec.DisableIntrospection {
 			return nil, errors.New("introspection disabled")
 		}
-		return introspection.WrapSchema(parsedSchema), nil
+		return introspection.WrapSchema(ec.Schema()), nil
 	}
 
 	func (ec *executionContext) introspectType(name string) (*introspection.Type, error) {
 		if ec.DisableIntrospection {
 			return nil, errors.New("introspection disabled")
 		}
-		return introspection.WrapTypeFromDef(parsedSchema, parsedSchema.Types[name]), nil
+		return introspection.WrapTypeFromDef(ec.Schema(), ec.Schema().Types[name]), nil
```

Comment:
I reckon line 277 and 284 have better coding style no matter we decide to make executableSchema configurable or not

Reply:
Avoiding global variables is good, but `ec.Schema().Types[name]` violates the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).

If any of `ec`, `ec.Schema()`, or `ec.Schema().Types` are nil, this will have a fatal error (panic due to nil pointer dereference) that is difficult to figure out which of these happened.

A temporary local variable might be more verbose, but in the event of a nil pointer dereference, the line number would then tell you immediately which of those was problematic.